doc/README.md

```diff
@@ -55,13 +56,13 @@ Ubuntu 16.04:
 ```bash
 ./doc.pl --out=html --include=user-guide --no-cache --var=os-type=debian --var=os-image=ubuntu:16.04 --var=package=test/package/pgbackrest_2.08-0_amd64.deb
 ```
-CentOS/RHEL 7:
+RHEL/CentOS 7:
 ```bash
-./doc.pl --out=html --include=user-guide --no-cache --var=os-type=centos7 --var=package=test/package/pgbackrest-2.08-1.el7.x86_64.rpm
+./doc.pl --out=html --include=user-guide --no-cache --var=os-type=rhel --var=os-image=centos:7  --var=package=test/package/pgbackrest-2.08-1.el7.x86_64.rpm
 ```
 CentOS/RHEL 8:
```

Comment:
Shouldn't `CentOS/RHEL 8:` have been changed to `RHEL/CentOS 8:`?

Reply:
Fixed in 365a558c089.

Reply:
And again in 1eb931b8ec99.